### ISSUE TITLE
Can't open directory containing broken symlink

### ISSUE THREAD

**Issue by cben** [tagspaces/tagspaces#182]
Tagspaces desktop app (ubuntu, 64 bit) refuses to switch view from ~/ to my ~/Videos directory, printing:
```
[10577:0108/140758:INFO:CONSOLE(707)] "Uncaught Error: ENOENT, no such file or directory '/home/beni/Videos/Elysium_trailer_1-4K-HDTN.mp4'", source: fs.js (707)
```
That file is a broken symlink [resulting from an incomplete git-annex sync].

Similarly it shows nothing if I add ~/Videos as a location.

**Comment by uggrock** [comment]
This is a known issue I am searching for a solution....

**Comment by uggrock** [comment]
Fixed in 1.10.0
